scripts/geopmpy/launcher.py

```diff
@@ -1078,8 +1078,20 @@ def reservation_option(self):
             result = ['--reservation', self.reservation]
         return result
 
+    do_warning_once = True
     def performance_governor_option(self):
-        return ['--cpu-freq=Performance']
+        result = []
+        governor_file = '/sys/devices/system/cpu/cpu0/cpufreq/scaling_governor'
+        with open(governor_file) as f:
```

Comment:
This is opening the governor file on the node where geopmlaunch is being executed.  I think we need to do something to run on the compute node, like when we create the topo, a launcher is used to do that.  Could be more of a circular dependency in this case, since you need the use launcher to figure out how the launcher should be configured.

Reply:
doh